Block-Logic/bl-rails-template
# 185709189 tweak configuration

#### What's this PR do?
- Tweaks various configuration files, see the list in PT

#### How to manually test it?
- Download master.key and development.key from keybase
- Run ./bin/setup or rails db:seed & check seed result, sample user and opt out request should be added to your db
- Run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/185709189)

#### Task completed checklist:

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
